bin/plugin/config.js

```diff
@@ -0,0 +1,17 @@
+const gitRepoOwner = 'WordPress';
+
+const config = {
+	slug: 'gutenberg',
+	name: 'Gutenberg',
+	team: 'Gutenberg Core',
+	githubRepositoryOwner: gitRepoOwner,
+	githubRepositoryName: 'gutenberg',
+	pluginEntryPoint: 'gutenberg.php',
+	buildZipCommand: '/bin/bash bin/build-plugin-zip.sh',
+	wpRepositoryReleasesUrl:
+		'https://github.com/WordPress/gutenberg/releases/tag/',
+	gitRepositoryUrl: 'https://github.com/' + gitRepoOwner + '/gutenberg.git',
+	svnRepositoryUrl: 'https://plugins.svn.wordpress.org/gutenberg',
```

Comment:
```suggestion
	wpRepositoryReleasesURL:
		'https://github.com/WordPress/gutenberg/releases/tag/',
	gitRepositoryURL: 'https://github.com/' + gitRepoOwner + '/gutenberg.git',
	svnRepositoryURL: 'https://plugins.svn.wordpress.org/gutenberg',
```

URL is an acronym

https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/#abbreviations-and-acronyms